README.md

```diff
@@ -378,7 +499,11 @@ While this document doesn't aim to critically assess the precision of these esti
 
 ### Concurrency
 
-What should happen if two tabs depend on the same file, check COS, see the file is not in COS, and start downloading? Should this be handled smartly? How often does this happen in practice? In the worst case, the file gets downloaded twice, but would then still only be stored once.
+What should happen if two tabs depend on the same file, check COS, see the file is not in COS, and start downloading? Should this be handled smartly? How often does this happen in practice? In the worst case, the file gets downloaded twice, but would then still only be stored once in COS. Our current thinking is to not deal with this special case based on the tolerable worst case.
+
+### Partial COS matches
+
+If the developer wants to check if two files A and B, with the hashes hash_A and hash_B are stored in COS, but only one of the two is stored, the API will still fail with a `NotFoundError` `DOMException` without revealing the partial match. Should it? Our current thinking is that it complicates error handling, especially since the expected use cases commonly require all files to be present for the app to function, for example, the tokenizer, configurations, weights, and graph with an AI model.
```

Comment:
Also, for privacy reasons. This would allow (very limited) enumeration of store contents.

Reply:
Addressed in https://github.com/tomayac/cross-origin-storage/pull/30/commits/661951110970861aad14372e528a82256ae6f36f.